bika/lims/content/abstractbaseanalysis.py

```diff
@@ -995,14 +995,6 @@ def getCategoryUID(self):
             return category.UID()
 
     @security.public
-    def getDepartmentTitle(self):
-        """Used to populate catalog values
-        """
-        department = self.getDepartment()
-        if department:
-            return department.Title()
-
-    @security.public
```

Comment:
Instead of removing, please use deprecated annotations. We need back-wards compatibility:

```python
@deprecated('[1.1] Use getDepartment().Title() instead')
def getDepartmentTitle(self):
    department = self.getDepartment()
    if department:
        return department.Title()
```